src/Microsoft.VisualStudio.Web.CodeGeneration.ProjectInfo/Dependency.cs

```diff
@@ -0,0 +1,37 @@
+﻿using System;
+using System.Collections.Generic;
+
+namespace Microsoft.VisualStudio.Web.CodeGeneration.ProjectInfo
+{
+    public class Dependency
+    {
+        public Dependency(string name, string version, string itemSpec)
+        {
+            Requires.NotNullOrEmpty(name);
+            Requires.NotNullOrEmpty(itemSpec);
+            Name = name;
+            Version = version;
+            ItemSpec = itemSpec;
+        }
+
+        public string Name { get; }
+        public string Version { get; }
+        public string ItemSpec { get; }
```

Comment:
This is an identifier for the 'item' in the output of the `ResolvePackageDependenciesDesignTime` target. 
This is not really used after creation, so will remove it.